README.md

```diff
@@ -187,18 +187,21 @@ More details about the docker images (like GPU support) can be found [here](http
 
 ### 🐍 Python API
 
+#### Running a multi-speaker and multi-lingual model
+
 ```python
+import torch
 from TTS.api import TTS
 
-# Running a multi-speaker and multi-lingual model
+# Get device
+device = "cuda" if torch.cuda.is_available() else "cpu"
 
 # List available 🐸TTS models and choose the first one
-model_name = TTS.list_models()[0]
+model_name = TTS().list_models()[0]
```

Comment:
This code snippet no longer works because `list_models` is no longer a static method (change introduced in https://github.com/coqui-ai/TTS/commit/3a104d5c49e6f9063e32ba5fed8e34f1dce25887). We need to instantiate a TTS object.